src/tests/test_transform.py

```diff
@@ -96,23 +95,17 @@ def test_isAddonNewer(self):
 		newAddon.lastTestedVersion = V_2020_3
 		newAddon.pathToData = "new-path"
 
-		self.assertDictEqual(getLatestAddons([oldAddon, newAddon], nvdaAPIVersions), {
-			# 1st addon is added because: oldAddon.addonId not in addons
-			# 2nd addon it not added: pass as incompatible
-			V_2020_2: {"stable": {"foo": oldAddon}, "beta": {}},
-			# 1st addon is added because: oldAddon.addonId not in addons
-			# 2nd addon updates it because: newAddon.addonVersion > oldAddon.addonVersion
-			V_2020_3: {"stable": {"foo": newAddon}, "beta": {}},
+		addonGenerator = (addon for addon in (oldAddon, newAddon))
```

Comment:
I don't think the generator does anything to aid readability here, why use it over a list in this case?

Reply:
It guarantees the order that the addons are consumed, and better replicates how the function is used internally.
I don't think it's super important though, as the function should support both datastructures as iterators.

Reply:
Not really, the method could have it's first lines:
```
addons = list(addons)
addons.reverse()
```